app/controllers/concerns/filterable_users.rb

```diff
@@ -0,0 +1,17 @@
+module FilterableUsers
+  extend ActiveSupport::Concern
+
+  private
+
+  def users_filter(users, params)
+    filter_by_email(users, params)
+  end
+
+  def filter_by_email(users, params)
+    if params[:query] && params[:query][:email].present?
```

Comment:
this is for encapsulating filter field names

Reply:
Why not use dig?

Reply:
I don't used this method before, but it better solution here.